cmd/root.go

```diff
@@ -28,6 +28,7 @@ import (
 	"strings"
 	"syscall"
 
+	"github.com/Traackr/binnacle/config"
```

Comment:
@jmmk Why do you need to import this?

Reply:
@dchancogne see below where the functions were moved to this file `func syncRepositories(repos []config.RepositoryConfig, args ...string) error {` uses `config.RepositoryConfig`